module/applications/activity/activity-sheet.mjs

```diff
@@ -399,6 +419,25 @@ export default class ActivitySheet extends Application5e {
 
   /* -------------------------------------------- */
 
+  /**
+   * Handle creating a new active effect and adding it to the applied effects list.
+   * @this {ActivityConfig}
+   * @param {Event} event         Triggering click event.
+   * @param {HTMLElement} target  Button that was clicked.
+   */
+  static async #addEffect(event, target) {
+    const effectData = {
+      name: this.item.name,
+      icon: this.item.img,
+      origin: this.item.uuid,
```

Comment:
It seems a bit strange to set the origin to the same Item that the ActiveEffect is already on, no?

Reply:
Well that's what we do when creating effects on items at the moment, though presumable this will be set again when the effect is applied through the chat message.

Reply:
Ah, I guess so that it's set to the correct Item if you drop it onto another one.